### PR TITLE
2019.01.22 onsuk

### PR DESCRIPTION
**Pull request by onsuk**
- [x] [leetcode] 976-largest-perimeter-triangle
- [ ] [leetcode] 114-flatten-binary-tree-to-linked-list
- [ ]  [leetcode] 121-best-time-to-buy-and-sell-stock

### CODE REVIEW

File: Leetcode/976-largest-perimeter-triangle/largest-perimeter-triangle.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class Solution:
+    def largestPerimeter(self, A):
+        A.sort()
+        A.reverse()
+        for _ in range(len(A)):
+            while((len(A)>2)):
+                if A[0] < (A[1]+A[2]):
+                    return sum(A[:3])
+                else:
+                    del A[0]
```
Comment (SeungUkLee) [inline Leetcode/976-largest-perimeter-triangle/largest-perimeter-triangle.py]: 이중 루프말고 루프 한번만 돌리고 `del` 사용안하고 어떻게 할 수 있을지 생각해봐봐

File: Leetcode/976-largest-perimeter-triangle/largest-perimeter-triangle.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class Solution:
+    def largestPerimeter(self, A):
+        A.sort()
+        A.reverse()
```
[the remark this answers is not in the document]
Comment (onsuk) [reply Leetcode/976-largest-perimeter-triangle/largest-perimeter-triangle.py]: wow thx! 🙂

[the remark this answers is not in the document]
Comment (onsuk) [reply Leetcode/976-largest-perimeter-triangle/largest-perimeter-triangle.py]: (미국 감성)

File: Leetcode/976-largest-perimeter-triangle/largest-perimeter-triangle.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+        for _ in range(len(A)):
+            while((len(A)>2)):
```
[the remark this answers is not in the document]
Comment (onsuk) [reply Leetcode/976-largest-perimeter-triangle/largest-perimeter-triangle.py]: such a stupid me...!! XD

[the remark this answers is not in the document]
Comment (SeungUkLee) [reply Leetcode/976-largest-perimeter-triangle/largest-perimeter-triangle.py]: :D

File: Leetcode/976-largest-perimeter-triangle/largest-perimeter-triangle.py (hunk 1 above)
Comment (onsuk) [reply Leetcode/976-largest-perimeter-triangle/largest-perimeter-triangle.py]: i've suddenly realized that it still works with deletion of for loop.
And replace `del A[0]` with `A.pop(0)`

Comment (SeungUkLee) [reply Leetcode/976-largest-perimeter-triangle/largest-perimeter-triangle.py]: okay :)

**Pull request merged**